src/mcu.cpp

```diff
@@ -1351,6 +1351,26 @@ int main(int argc, char *argv[])
             {
                 resetType = ResetType::GM_RESET;
             }
+            else if (!strcmp(argv[i], "-h") || !strcmp(argv[i], "-help") || !strcmp(argv[i], "--help"))
+            {
+                printf("Usage: %s [options]\n", argv[0]);
+                printf("Options:\n");
+                printf("  -h, -help, --help              Display this information.\n");
+                printf("\n");
+                printf("  -p:<port_number>               Set MIDI port.\n", argv[0]);
+                printf("  -a:<device_number>             Set Audio Device index.\n", argv[0]);
+                printf("  -ab:<page_size>:[page_amount]  Set Audio Buffer size.\n", argv[0]);
+                printf("\n");
+                printf("  -mk2                           Use MK2 ROM set.\n");
```

Comment:
`SC-55mk2`, `SC-55mk1`, `SC-55st`, `CM-300/SCC-1` and `JV-880` would be preffered names